src/main/resources/api.yaml

```diff
@@ -87,6 +87,112 @@ paths:
           description: User is not authorized
         404:
           description: Project not found
+  /projects/{id}/users:
+    get:
+      tags:
+        - projects
+      summary: Retrieves users belonging to the project
+      operationId: getUsersFromProject
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: id
+          description: Project ID
+          required: true
+          type: integer
+          format: int64
+      responses:
+        200:
+          description: Users retrieved successfully
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/UserInProjectDto'
+        404:
+          description: Project not found
+    post:
```

Comment:
Dodawanie miało się odbywać po adresie email 
Oprócz tego, fajna byłaby metoda, która po podaniu np. `adam.kow` zwróci listę np. 5 maili z bazy, które pasują (do textboxa z podpowiedziami [ale to taki na razie niepotrzebny ficzer])

Reply:
Czemu akurat po mailu? Wszędzie jest używane user ID.

Reply:
W sensie w widoczku się dodaje po mailu? O to chodzi?

Reply:
Noo, tak jest na jirze jeśli mnie pamięć nie myli

Reply:
Znaczy dodawać kogoś do projektu na froncie po id z bazy to tak chyba średnio?

Reply:
W sensie tutaj searchbox byłby fajny, ale bez przesady

Reply:
Na razie bym zakładała podanie poprawnego maila

Reply:
> Znaczy dodawać kogoś do projektu na froncie po id z bazy to tak chyba średnio?

Tzn? User będzie wpisywać email, front wysyła zapytanie o userów z tą częścią maila wraz z ich nazwami i ID a potem przekazuje ID usera do dodania którego wybrał użytkownik, nie tak?

Reply:
Znaczy no tak, brzmi pięknie, ale `/user` jeszcze kompletnie nie mamy i paru innych rzeczy też nie, a front nie jest jakoś na tyle skończony, żeby serio robić wyszukiwanie

Reply:
Jak zostanie nam czasu przed 10.05, to pewnie